examples/pytorch/graphsaint/README.md

```diff
@@ -0,0 +1,109 @@
+# GraphSAINT
+
+This DGL example implements the paper: GraphSAINT: Graph Sampling Based Inductive Learning Method.
+
+Paper link: https://arxiv.org/abs/1907.04931
+
+Author's code: https://github.com/GraphSAINT/GraphSAINT
+
+Contributor: Liu Tang ([@lt610](https://github.com/lt610))
+
+## Dependencies
+
+- Python 3.7.0
+- PyTorch 1.6.0
+- NumPy 1.19.2
+- Scikit-learn 0.23.2
+- DGL 0.5.3
+
+## Dataset
+
+All datasets used are provided by Author's [code](https://github.com/GraphSAINT/GraphSAINT). They are available in [Google Drive](https://drive.google.com/drive/folders/1zycmmDES39zVlbVCYs88JTJ1Wm5FbfLz) (alternatively, [Baidu Wangpan (code: f1ao)](https://pan.baidu.com/s/1SOb0SiSAXavwAcNqkttwcg#list/path=%2F)). Dataset summary("m" stands for multi-label classification, and "s" for single-label.):
+| Dataset | Nodes | Edges | Degree | Feature | Classes | Train/Val/Test |
+| :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| PPI | 14,755 | 225,270 | 15 | 50 | 121(m) | 0.66/0.12/0.22 |
+| Flickr | 89,250 | 899,756 | 10 | 500 | 7(s) | 0.50/0.25/0.25 |
+| Reddit | 232,965 | 11,606,919 | 50 | 602 | 41(s) | 0.66/0.10/0.24 |
+| Yelp | 716,847 | 6,877,410 | 10 | 300 | 100(m) | 0.75/0.10/0.15 |
+| Amazon | 1,598,960 | 132,169,734 | 83 | 200 | 107(m) | 0.85/0.05/0.10 |
+
+## Minibatch training
+
+Run with following:
+```bash
+python train_sampling.py --gpu 0 --dataset ppi --sampler node --node-budget 6000 --num-repeat 50 --n-epochs 1000 --n-hidden 512 --arch 1-0-1-0 --batch-norm --dropout 0.1
```

Comment:
"0.1" -> "0.0"? See https://github.com/GraphSAINT/GraphSAINT/blob/master/train_config/table2/ppi2_n.yml

Reply:
I also tried dropout=0.0, but the accuracy will drop by one point

Reply:
Ok, then let's keep 0.1.